js/src/mlt-vector-tile-js/VectorTileLayer.ts

```diff
@@ -0,0 +1,25 @@
+import { VectorTileFeature } from './VectorTileFeature';
+
+class VectorTileLayer {
+  version: number;
+  name: string | null;
+  extent: number;
+  length: number = 0;
+
+  _raw: any;
+  _keys: string[];
+  _values: any[];
+  _features: VectorTileFeature[] = [];
+
+  constructor(layer) {
+    this.name = layer.name;
+    this._features = layer.features.map((feature) => new VectorTileFeature(feature));
```

Comment:
I recommend avoiding this `map` and rather lazily creating a `new VectorTileFeature` below in the `feature(i)` call like vector-tile-js: https://github.com/mapbox/vector-tile-js/blob/77851380b63b07fd0af3d5a3f144cc86fb39fdd1/lib/vectortilelayer.js#L60

Reply:
Done!